### ISSUE TITLE
Attest.go Error:  String method: reflect.Value.Interface: cannot return value obtained from unexported field or method

### ISSUE THREAD

**Issue by taariq** [palomachain/paloma#614]
# What is happening?
<details>
  <summary>Section description</summary>
<i>Provide as much context as you can. Give as much context as you can to make it easier for the developers to figure what is happening.</i>
</details>

Printing attest.go returns the error in logs for this line: https://github.com/palomachain/paloma/blob/master/x/consensus/keeper/attest.go#L29

`MESSAGE.string:+%!v(PANIC=String method: reflect.Value.Interface: cannot return value obtained from unexported field or method)`. 

Issue: We are unable to determine where the reflect method was used here and what unexported filed from the transaction message is causing the failure.

# Paloma and pigeon versions and logs
<details>
  <summary>Section description</summary>
<i>Write down paloma version.</i>
<i>Write down pigeon version.</i>
<i>Copy and paste pigeon config file as well as relevant ENV variables.</i>
</details>

Current Master on Paloma and Pigeon.

# How to reproduce?
<details>
  <summary>Section description</summary>
  <i>Please write detailed steps of what you were doing for this bug to appear.</i>
</details>

Deploy contract on ETH with Paloma. Paloma fails to attest to deployment tx.

# What is the expected behaviour?
<details>
  <summary>Section description</summary>
<i>If you know, please write down what is the expected behaviour. If you don't know, that's ok. We can have a discussion in comments.</i>
</details>

EIP-1559 deployments are successful.

**Comment by taariq** [comment]
Tracking with #615 changes.

**Comment by taariq** [comment]
@alexanderbez we identify that the panic occurs in this function https://github.com/palomachain/paloma/blob/master/x/consensus/keeper/concensus_keeper.go#L110

where `msgs` appears to have an unexported field/method, but we are unable to determine how to find this. Does this clarify?

**Comment by taariq** [comment]
@alexanderbez also, from our logs, we have the following linies:
```
Jan 03 21:09:16 testnet-1 palomad[4096867]: {"level":"info","module":"x/evm","chain-id":"eth-main","queue":"evm/eth-main/evm-turnstone-message","time":"2023-01-03T21:09:16Z","message":"supported-queues"}
Jan 03 21:09:16 testnet-1 palomad[4096867]: {"level":"info","module":"x/evm","chain-id":"eth-main","queue":"evm/eth-main/validators-balances","time":"2023-01-03T21:09:16Z","message":"supported-queues"}
Jan 03 21:09:16 testnet-1 palomad[4096867]: {"level":"info","module":"x/evm","chain-id":"eth-main","queue":"evm/eth-main/collect-fund-events","time":"2023-01-03T21:09:16Z","message":"supported-queues"}
Jan 03 21:09:16 testnet-1 palomad[4096867]: [GetMessagesFromQueue] msgs: 0, %!v(PANIC=String method: reflect.Value.Interface: cannot return value obtained from unexported field or method)
```

Before the panic `collect-fund-events` runs without any issues and returns a message. It's declared here:
https://github.com/palomachain/paloma/blob/master/x/evm/keeper/keeper.go#L36

However, SignaturePrefix here:
https://github.com/palomachain/paloma/blob/master/x/evm/keeper/keeper.go#L37

Seems to be the data that is failing with the panic. We're trying to confirm this is the failure that is leading the panic. Does this clarify?

**Comment by taariq** [comment]
Fixed with #618 which creates custom type for QuedMessagesI interface.
